addons/website_sale/static/src/js/location_selector/map/map.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <templates xml:space="preserve">
     <t t-name="website_sale.locationSelector.map">
-        <div class="w-100 h-100" style="width: 30rem! important; height: 30rem !important;" t-ref="mapContainer"></div>
+        <div class="w-100 flex-grow-1" t-ref="mapContainer"/>
     </t>
 
     <t t-name="website_sale.locationSelector.map.marker">
-        <div style="color: var(--primary);">
-            <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" viewBox="0 0 61 78.9" style="enable-background:new 0 0 61 78.9;" xml:space="preserve">
+        <div style="color: var(--light);"> <!-- todo vcr change var to primary on click -->
```

Comment:
on click? do you mean when the location is selected?

Reply:
Not possible IMO, the marker shall all have the same design.

Reply:
So it's not possible to change the color of the selected marker?
![Capture d’écran 2024-04-26 à 16 22 07](https://github.com/odoo-dev/odoo/assets/80679690/b7525abd-b205-4a1d-a960-ae532db68705)